Add-README-files-3/8 (#5819)
TheAlgorithms/Python: data_structures/README.md

# Data Structures

> There exist [many](https://en.wikipedia.org/wiki/List_of_data_structures) types of data structures, but some structures are more useful to keep some type of data over another type.

Reviewer: ```suggestion
[Many](https://en.wikipedia.org/wiki/List_of_data_structures) types of data structures exist, but some structures are more useful than others when storing certain types of data.
```

> The types used everywhere when programming is the boolean, the integer, the character and the floating point number. Those can lead to more advanced type, such as list, array, set, tuple, stack etc.

Reviewer: This line is wordy and doesn't make sense

(If this subject has your interest, you should do your own research because it is a vast subject. As often, the starting point should be the Wikipedia page.)

* <https://en.wikipedia.org/wiki/Data_structure>

> * <https://www.tommyds.it/doc/benchmark.html>

Reviewer: Doesn't seem relevant
